ide/css.editor/external/css3-spec-howtocreate.txt

```diff
@@ -45,15 +45,16 @@ http://www.w3.org/TR/css3-preslev/ \
 http://www.w3.org/TR/css3-regions/ \
 http://www.w3.org/TR/css3-ruby/ \
 http://www.w3.org/TR/css3-speech/ \
-http://www.w3.org/TR/css3-text/ \
 http://www.w3.org/TR/css3-transitions/ \
 http://www.w3.org/TR/css3-ui/ \
 http://www.w3.org/TR/css3-values/ \
 http://www.w3.org/TR/css3-writing-modes/ \
 http://www.w3.org/TR/css-align-3/ \
 http://www.w3.org/TR/css-break-3/ \
 http://www.w3.org/TR/css-position-3/ \
-http://www.w3.org/TR/css-sizing-3/
+http://www.w3.org/TR/css-sizing-3/ \
+http://www.w3.org/TR/css-contain-3/ \
+http://www.w3.org/TR/css-text-4/
```

Comment:
What about uploading the newly generated `css3-spec.zip` with a version number? This could be `css3-spec-2023-09-25.zip`, for instance. This will avoid breaking previous builds, right?

Reply:
The build is not broken by this (see above), the version number will more or less be an arbitrary number and this in my mind the SHA1 sum is as good as a number I would imagine. Does this address your concerns?
Oh, it might be helpful to know, that this update five times now since the Apache transition, all in this way.